src/App.tsx

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
```

Comment:
I think, you can use `interface `here instead of `type`

Reply:
Yes, use interfaces to describe objects